db/version_set.cc

```diff
@@ -1697,6 +1701,58 @@ void VersionStorageInfo::GenerateLevel0NonOverlapping() {
   }
 }
 
+void VersionStorageInfo::UpdateBottommostFiles() {
```

Comment:
Nit: This is not updating a bottommost file list, it's actually generating it, right?

Reply:
Yes that sounds right.